src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -59,6 +59,9 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   /** The in-memory list of Users. */
   private List<User> users;
 
+  /** The in-memory list of admins. */
+  private List<User> admins;
```

Comment:
This variable is never accessed anywhere. What is its purpose? I think you need to set it by calling loadAdmins from persistentStorageAgent